### PR TITLE
BleCommandQueue : fix second to millisecond

### PR DESCRIPTION
**Pull request by sandros06**

### CODE REVIEW

**Comment by lothrop** [comment]
This would break a lot of existing implementations.

**Comment by sandros06** [comment]
Actually the default timeout is 10 millisecond very short ! 

Becaus variable name not good or missing *1000

File: Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs [hunk 1]
```diff
@@ -82,10 +82,10 @@ public class BleCommand<T> : IBleCommand
 
         public int TimeoutInMiliSeconds { get; }
 
-        public BleCommand(Func<Task<T>> taskSource, int timeoutInSeconds)
+        public BleCommand(Func<Task<T>> taskSource, int timeoutInMiliSeconds)
         {
             _taskSource = taskSource;
-            TimeoutInMiliSeconds = timeoutInSeconds;
```
Comment (sandros06) [inline Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs:88]: Typo here

File: Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs [hunk 2]
```diff
@@ -18,9 +18,9 @@ public BleCommandQueue()
 
         }
 
-        public Task<T> EnqueueAsync<T>(Func<Task<T>> bleCommand, int timeOutInSeconds = 10)
+        public Task<T> EnqueueAsync<T>(Func<Task<T>> bleCommand, int timeoutInMiliSeconds = 10000)
```
Comment (Hoidi) [inline Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs:21]: Change all MiliSeconds to Milliseconds

**Review by Hoidi — CHANGES_REQUESTED** [review]

File: Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs [hunk 3]
```diff
@@ -18,9 +18,9 @@ public BleCommandQueue()
 
         }
 
-        public Task<T> EnqueueAsync<T>(Func<Task<T>> bleCommand, int timeoutInMiliSeconds = 10000)
+        public Task<T> EnqueueAsync<T>(Func<Task<T>> bleCommand, int timeoutInMiliseconds = 10000)
```
Comment (Hoidi) [inline Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs:21]: Needs one more L in Milli -> Milliseconds

**Review by Hoidi — CHANGES_REQUESTED** [review]

File: Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs [hunk 4]
```diff
@@ -99,7 +99,7 @@ public async Task ExecuteAsync()
             {
                 IsExecuting = true;
                 var source = _taskSource();
-                if (source != await Task.WhenAny(source, Task.Delay(TimeoutInMiliSeconds)))
+                if (source != await Task.WhenAny(source, Task.Delay(TimeoutInMiliseconds)))
```
Comment (janusw) [inline Source/Plugin.BLE.Abstractions/Utils/BleCommandQueue.cs:102]: Another typo here, as detected by the [GH Actions build](https://github.com/xabre/xamarin-bluetooth-le/pull/533/checks?check_run_id=3393160046):

```
BleCommandQueue.cs(102,69): error CS0103: The name 'TimeoutInMiliseconds' does not exist in the current context
```

Have you ever even tried to compile this branch?!?

**Pull request closed without merging**